services/brig/test/integration/API/Calling.hs

```diff
@@ -75,6 +86,26 @@ testCallsConfigMultiple b turnUpdater = do
   let _expected = List1.singleton (toTurnURILegacy "127.0.0.1" 3478)
   modifyAndAssert b uid getTurnConfigurationV1 turnUpdater "turn:127.0.0.1:3478" _expected
 
+testSFT :: Brig -> Opts.Opts -> Http ()
+testSFT b opts = do
+  uid <- userId <$> randomUser b
+  cfg <- getTurnConfigurationV2 uid b
+  liftIO $
+    assertEqual
+      "when SFT discovery is not enabled, sft_servers shouldn't be returned"
+      Nothing
+      (cfg ^. rtcConfSftServers)
+  withSettingsOverrides (opts & Opts.sftL ?~ Opts.SFTOptions "integration-tests.zinfra.io" Nothing (Just 0.001)) $ do
+    cfg1 <- retryWhileN 10 (isNothing . view rtcConfSftServers) (getTurnConfigurationV2 uid b)
+    -- These values are controlled by https://github.com/zinfra/cailleach/tree/77ca2d23cf2959aa183dd945d0a0b13537a8950d/environments/dns-integration-tests
```

Comment:
it's not ideal that we're having another link into a private repo here, but not linking it would be worse IMO, so :+1:

Reply:
Yeah, I chose lesser of the two evils. I was thinking of committing that code in this repo, but decided to not invent yet another way to do infra (and make a third evil?).